wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,202 @@
+- `F4`: Stop playing and reset duration to 0 (jukebox)
+- `V` or `F5`: Next song (jukebox)
+- `J` or `F6`: Jump To (jukebox)
+- `R`: Random song (jukebox)
+- `D`: osu!direct ([osu!supporters](/wiki/osu!supporter) only)
+- `O`: Options
+- `Q` or `Esc`, then `Esc`: Quit (displays a prompt)
+
+## Song Select
+
+- `F1`: Display [mod](/wiki/Game_Modifiers) selection screen
+- `F2`: Random map selection
+- `Shift` + `F2`: Undo random map selection
+- `F3`: Display beatmap options
+- `F5`: Refresh listing (repair mode)
+- `Shift` + `Delete`: Delete current map
+- `Ctrl` + `1`: Switch to [osu!](/wiki/osu!)
+- `Ctrl` + `2`: Switch to [osu!taiko](/wiki/osu!taiko)
+- `Ctrl` + `3`: Switch to [osu!catch](/wiki/osu!catch)
+- `Ctrl` + `4`: Switch to [osu!mania](/wiki/osu!mania)
+- `Ctrl` + `Enter`: Play current map with [Auto](/wiki/Auto) mod
+- `Ctrl` + `Shift` + `Enter`: Play current map with [Cinema](/wiki/Cinema) mod
+- `Ctrl` + `A`: Toggle Auto mod
+- `Ctrl` + `F3` or `Ctrl` + `+`: Increase osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `F4` or `Ctrl` + `-`: Decrease osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `Backspace`: Delete last word (when using search)
+- `Up` or `Down`: Change selection
+- `Left` or `Right`: Change beatmap set
+- `Page Up` or `Page Down`: Change selection (large)
+- `Enter`: Make selection
+- `Shift` + `Left Arrow` or `Shift` + `Right`: Change group
+- `Shift` + `Enter`: Open/close group
+
+## Mods screen
+
+**Notes:**
+
+- The key references below assumes that none of the mods are selected.
+- Some mods may require the player to press the key twice.
+
+- `Q`: [Easy](/wiki/Easy_mod)
+- `W`: [No Fail](/wiki/No_Fail)
+- `E`: [Half Time](/wiki/Half_Time)
+
+---
+
+- `A`: [Hard Rock](/wiki/Hard_Rock)
+- `S`: [Sudden Death](/wiki/Sudden_Death)
+- `S`, then `S`: [Perfect](/wiki/Perfect)
+- `D`: [Double Time](/wiki/Double_Time)
+- `D`, then `D`: [Nightcore](/wiki/Nightcore)
+- `F`: [Hidden](/wiki/Hidden) (osu!standard, osu!taiko, and osu!catch)
+- `F`: [Fade In](/wiki/Fade_In) (osu!mania only)
+- `F`, then `F`: [Hidden](/wiki/Hidden) (osu!mania only)
+- `G`: [Flashlight](/wiki/Flashlight)
+
+---
+
+- `X`: [Relax](/wiki/Relax)
+- `C`: [Auto Pilot](/wiki/Auto_Pilot)
+- `V`: [Spun Out](/wiki/Spun_Out)
+- `B`: Auto
+- `B`, then `B`: Cinema
+
+---
+
+- `3`: [3K](/wiki/3K) (osu!mania only)
+- `4`: [4K](/wiki/4K) (osu!mania only)
+- `5`: [5K](/wiki/5K) (osu!mania only)
+- `6`: [6K](/wiki/6K) (osu!mania only)
+- `7`: [7K](/wiki/7K) (osu!mania only)
+- `8`: [8K](/wiki/8K) (osu!mania only)
+- `9`: [9K](/wiki/9K) (osu!mania only)
+
+---
+
+- `1`: Deselect selected mods
+- `2`: Closes the mod selection dialog
+
+## Play Mode
+
+- `+` or `-`: Adjust local offset (hold `Alt` to change the precision).
+- `Esc`: Pause or quit song if Auto or Cinema mod is used.
+- `Space`: Skip opening cut-scene.
+- `Tab`: Toggle scoreboard
+- `Shift` + `Tab`: Toggle in-game interface
+- `F3` or `F4`: Adjust speed for osu!mania.
```

Comment:
This only works at the start of songs for me, worth mentioning or did I break something?